tests/integrations/wsgi/test_wsgi.py

```diff
@@ -30,3 +51,15 @@ def test_basic(sentry_init, crashing_app, capture_events):
         "query_string": "",
         "url": "http://localhost/",
     }
+
+
+def test_env_modifing_(sentry_init, crashing_env_modifing_app, capture_events):
```

Comment:
Could you fix this name to something that doesn't have an underscore at the end?

Reply:
Sure, sorry I missed that.

Reply:
There supposed to be `_app`